src/index.js

```diff
@@ -114,7 +113,6 @@ const App = () => {
         return todos;
     }
 
-    // does this count as storing the value instead of executing the function in props?
     const sortedTodos = sortTodos(); 
```

Comment:
does this count as storing in a var instead of rendering a prop?

Reply:
Yes and this is much easier to read!